goaviatrix/distributed_firewalling_zero_trust_rule.go

```diff
@@ -0,0 +1,50 @@
+package goaviatrix
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+)
+
+type DistributedFirewallingZeroTrustRule struct {
+	Action  string `json:"action"`
+	Logging bool   `json:"logging"`
+	UUID    string `json:"uuid,omitempty"`
+}
+
+func (c *Client) UpdateDistributedFirewallingZeroTrust(ctx context.Context, request *DistributedFirewallingZeroTrustRule) error {
+	endpoint := "microseg/edit-zero-trust"
+	return c.PutAPIContext25(ctx, endpoint, request)
+}
+
+func (c *Client) GetDistributedFirewallingZeroTrustRule(ctx context.Context) (map[string]string, error) {
+	postRuleListUUID := "defa11a1-3000-5000-0000-000000000000"
+	zeroTrustRuleUUID := "defa11a1-0000-0000-0000-000000000000"
```

Comment:
Are these well-known constants? Could you link to a reference to help future maintainers?